shell.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs ? import <nixpkgs> {} }:
+	let
+	overrides = (builtins.fromTOML (builtins.readFile ./rust-toolchain.toml));
+	libPath = with pkgs; lib.makeLibraryPath [
+		# load external libraries that you need in your rust project here
+	];
```

Comment:
Is this copied from some template? It does nothing, you can remove it

Reply:
indeed it is. Could remove it but I thought to leave it to make future contributions and fixes easier (don't have to go searching for how to do X). Equally happy to remove it since functionally it doesn't do anything (yet)

Reply:
@yaleman Any preference re keeping it or removing it?

Reply:
Meh, leave it until the rest of the bike-shedding in this PR is done, if it’s not doing anything at that time it can be removed.